docs/Account/methods.md

```diff
@@ -0,0 +1,284 @@
+
+**Usage**
+
+```jsx
+let provider = new ethers.providers.JsonRpcProvider("rpcUrl");
+let signer = new ethers.Wallet("private key", provider);
+
+
+const smartAccount = await BiconomySmartAccountV2.create({
+  signer: signer,
+  chainId: ChainId.POLYGON_MUMBAI, // Specify the desired chain (e.g., Polygon Mumbai)
+  bundlerUrl: "", // bundler URL (required)
+  biconomyPaymasterApiKey: "", // Instance of the paymaster
+});
+
+```
+
+**Parameters**
+
+*required params are explicitly mentioned*
+
+- config (`object`, required): A `BiconomySmartAccountV2Config` object containing configuration options for creating the smart account.
+  - signer(`Signer`, required): The signer instance that will need to be passed. 
+  - chainId (`ChainId` enum, required): The identifier for the blockchain network. (e.g., ChainId.POLYGON_MUMBAI).
+  - bundlerUrl (`string`, required) OR bundler (`IBundler`, required) : bundler url which will be internally used to create bundler instance or the bundler instance. Bundler instance can also be used if one wants to customise the bundler. Refer to bundler [integration](./../Bundler/integration.mdx) for more details on bundler.
+  - biconomyPaymasterApiKey(`string`) OR paymaster (`IPaymaster`): one can either pass paymaster API key or the paymaster instance.
+  - entryPointAddress (`string`): default entry point address will be used if not passed, otherwise the passed address will be used.
+  - defaultValidationModule (`BaseValidationModule`): The default validation module to start with ( ECDSA ).
+  - activeValidationModule (`BaseValidationModule`): The active validation module to start with ( either ECDSA or Multi-chain ).
+  - rpcUrl (`string`): RPC URL of the chain
+  - index (`number`): index to create multiple smart accounts for an EOA
+
+**Returns**
+
+- `smartAccount` (`object`): An instance of the Biconomy Smart Account V2.
+
+:::info
+Building on Chiliz Mainnet or the Spicy Testnet? Note that the entry point address on this is different as it was deployed by us on the Biconomy team. The address of the entry point is : [0x00000061FEfce24A79343c27127435286BB7A4E1](https://scan.chiliz.com/address/0x00000061FEfce24A79343c27127435286BB7A4E1/contracts#address-tabs)
+:::
+
+## Smart Account Get Methods
+
+### getAccountAddress( )
+
+This method is used to retrieve the account address associated with the `smartAccount` instance.
+
+**Usage**
+
+```jsx
+const address = await smartAccount.getAccountAddress();
+```
+
+**Returns**
+
+- address (`Promise<string>`): A Promise resolving to the account address associated with the smartAccount instance.
+
+### getNonce( )
+
+This method is used to retrieve the nonce associated with the `smartAccount` instance.
+
+**Usage**
+
+```jsx
+
+const nonce = await smartAccount.getNonce();
+console.log(nonce.toNumber());
+```
+**Parameters**
+- nonceKey(`number`): one can also pass the optional parameter nonceKey in the case of two-dimensional nonces
```

Comment:
there is no example of how to pass nonceKey

it's usually, 

const nonceKey = 10;
const nonce = await biconomySmartAccount.getNonce(nonceKey);